### ISSUE TITLE
h5dump tfloatsattrs fails on Windows 10 + VS 2022 Community Edition

### ISSUE THREAD

**Issue by hyoklee** [HDFGroup/hdf5#1191]
```
The following tests FAILED:
	1657 - H5DUMP-tfloatsattrs (Failed)
Errors while running CTest
```

**Comment by byrnHDF** [comment]
Yes this has been failing for VS2019. It is a rounding difference. The trouble is that the version on github doesn't fail. Must be a VS version that determines which output will match.

**Comment by hyoklee** [comment]
@byrnHDF , can you add VS2022 to CI Actions?

**Comment by hyoklee** [comment]
d7466741eafcaf117818905a6cf2bcc2e798e2cc

**Comment by hyoklee** [comment]
Windows 11 reports the same failure. 
Let's leave it open for users until doc or release_docs mentions this issue.

**Comment by byrnHDF** [comment]
This test has been changed and should no longer fail.
